pinecone/db_data/import_error.py

```diff
@@ -1,62 +0,0 @@
-class IndexClientInstantiationError(Exception):
```

Comment:
I decided to nix these because it was causing a problem when generating docs. In the past, this was added to chastise people if they tried to instantiate the `Index` class without going through a parent client object such as `Pinecone`, but that small benefit is not worth having wonky looking documentation.